### PR TITLE
Display useful message

### LINKED ISSUE #2110: Replace log message in Product Photos by something more useful
image
The success message is very geeky.

### PR DESCRIPTION
**Pull request by PrajwalM2212**
fixes #2110

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -900,6 +900,7 @@
     <string name="set_img_front">Set Image as Front</string>
     <string name="report_img">Report Image</string>
     <string name="detected_ingredients">Detected Ingredients</string>
+    <string name="set_image_name">"Successfully set the image as "</string>
```
Comment (teolemon) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="set_image_name">Successfully set the image as </string>
```

Comment (PrajwalM2212) [reply app/src/main/res/values/strings.xml]: @teolemon I had to do that for adding the extra space adter `as`. The double quotes will not be rendered in the app.

**Comment by teolemon** [comment]
@ripetrescu your issue was solved. Looks good ?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (teolemon) [reply app/src/main/res/values/strings.xml]: Translators don't have to question themselves about double quotes @PrajwalM2212

Comment (PrajwalM2212) [reply app/src/main/res/values/strings.xml]: @teolemon I did not think about from that perspective but it is fixed now

**Pull request merged**